multi-node/aws/pkg/config/config.go

```diff
@@ -160,8 +162,10 @@ func (c Cluster) Config() (*Config, error) {
 	config.APIServers = fmt.Sprintf("http://%s:8080", c.ControllerIP)
 	config.SecureAPIServers = fmt.Sprintf("https://%s:443", c.ControllerIP)
 	config.APIServerEndpoint = fmt.Sprintf("https://%s", c.ExternalDNSName)
-	if config.UseCalico {
-		config.K8sNetworkPlugin = "cni"
+	config.K8sNetworkPlugin = "cni"
+
+	if config.ContainerRuntime == "rkt" && config.ReleaseChannel != "alpha" {
```

Comment:
Not sure if it's possible for us to introspect an actual release version this early (likely not). If it is difficult, ignore this as this is sufficient. But if we can target a release, that would be ideal.